buildscripts/README.md

```diff
@@ -84,13 +84,11 @@ When something goes wrong during the build, the log file will be launched (you c
 
 In addition to the log files in `C:\php-sdk`, you can examine the contents of `C:\php-sdk\phpsdk-build-task.bat`, which is overwritten every time you run the build scripts.
 
-#### Building the extensions unattended
+#### Variation
```

Comment:
I still don't like this name 'variation'... maybe 'Setting alternative destination directory' or something like that?